### PR TITLE
Update manager runtime configs. Bump carbon-coordination versions

### PR DESCRIPTION
**Pull request by Anoukh**
## Purpose
Update manager runtime with new configs for zookeeper configuration
Bump carbon-coordination to 2.0.16

### CODE REVIEW

File: modules/distribution/carbon-home/conf/manager/deployment.yaml [hunk 1]
```diff
@@ -290,4 +290,7 @@ deployment.config:
   datasource: SP_MGT_DB           # define a mysql datasource in datasources and refer it from here.
   minResourceCount: 1
   bootstrapURLs: localhost:9092   # kafka urls
-  zooKeeperURLs: localhost:2181   # zookeeper urls
+  zooKeeperConfig:
```
Comment (tishan89) [inline modules/distribution/carbon-home/conf/manager/deployment.yaml]: Don't we need to change bean classes to comply to this change?

Comment (Anoukh) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: Changes done in carbon-analytics#1233

Comment (Anoukh) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: Changes done in wso2/carbon-analytics#1233

Comment (this) [inline modules/distribution/carbon-home/conf/manager/deployment.yaml]: This change breaks backward compatibility with SP-4.0.0 & SP-4.1.0. Shall we try an approach that preserves backward compatibility.

Comment (Anoukh) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: The old config is not removed but deprecated so that nothing breaks.

Comment (tishan89) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: Let's also put a log so that user knows when they are using deprecated configs

Comment (Anoukh) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: Will add a warn log and update in carbon-analytics

Comment (this) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: We can add that log when we consume `zooKeeperConfig`. Check whether `zooKeeperURLs` is not null, and if so print a WARN log.

Comment (Anoukh) [reply modules/distribution/carbon-home/conf/manager/deployment.yaml]: Log added in wso2/cabon-analytics#1258

**Pull request merged**